### PR TITLE
docs(opentelemetry): rewrite

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Rewrite:
  - Use simple words
  - Use `must` to indicate requirements
  - Use `you` to pull the reader in
  - Use lists
  - Write short paragraphs
  - Write short sentences
  - End sentence with full stop
  - Add headings

## Context

Follow up after:

- https://github.com/renovatebot/renovate/pull/15731

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/opentelemetry.md [hunk 1]
```diff
@@ -9,24 +9,34 @@ description: How to use OpenTelemetry with Renovate
...
 
-Renovate can only send traces and only via the OpenTelemetryProtocol (OTLP), other observability data or transfer protocols are not supported.
+## Limitations
+
+Renovate only:
+
+- sends traces
+- uses the OpenTelemetryProtocol (OTLP)
+
+Renovate does not support other observability data or transfer protocols.
```
Comment (Churro) [inline docs/usage/opentelemetry.md:27]: Hope you don't mind if I chime in here but I think a small misunderstanding has sneaked in:

- While OTLP support traces, metrics, and logs, renovate currently only supports traces. 
- _Uses the OpenTelemetry Protocol_ (mind the space) isn't a limitation. I think the original sentence wanted to say that besides OTLP no other specifications for telemetry data ingestion are supported.
- _Transfer protocols_ is misleading. OTLP can send traces via HTTP (in JSON-encoded protobuf or binary-encoded protobuf format) or via gRPC. renovate uses [@opentelemetry/exporter-trace-otlp-http](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http) under the hood. This means, renovate sends traces via [OTLP/HTTP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp) in JSON-encoded protobuf format only. 

_Other observability data_ could be exemplified with stats on caching, error events, no. of found dep updates, etc.

Comment (HonkingGoose) [reply docs/usage/opentelemetry.md:27]:
```suggestion
## Limitations

While OTLP support traces, metrics, and logs, Renovate only supports traces.
This means Renovate does not support other observability data like: stats on caching, error events, number of found updates, and so on.

Besides OTLP no other specifications for telemetry data ingestion are supported.

Renovate uses [`@opentelemetry/exporter-trace-otlp-http`](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http) under the hood.
This means that Renovate sends traces via [OTLP/HTTP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp) in JSON-encoded protobuf format only.
```

File: docs/usage/opentelemetry.md [hunk 2]
```diff
@@ -9,24 +9,35 @@ description: How to use OpenTelemetry with Renovate
...
 - metrics
 - logs
 
-Renovate can only send traces and only via the OpenTelemetryProtocol (OTLP), other observability data or transfer protocols are not supported.
+## Limitations
+
+While OTLP support traces, metrics, and logs, Renovate only supports traces.
+This means Renovate does not support other observability data like: stats on caching, error events, number of found updates, and so on.
+
+Besides OTLP no other specifications for telemetry data ingestion are supported.
```
[the remark this answers is not in the document]
Comment (HonkingGoose) [reply docs/usage/opentelemetry.md:25]:
```suggestion
```

File: docs/usage/opentelemetry.md [hunk 3]
```diff
@@ -9,24 +9,34 @@ description: How to use OpenTelemetry with Renovate
...
 - traces
 - metrics
 - logs
 
-Renovate can only send traces and only via the OpenTelemetryProtocol (OTLP), other observability data or transfer protocols are not supported.
+## Limitations
+
+While OTLP support traces, metrics, and logs, Renovate only supports traces.
+This means Renovate does not support other observability data like: stats on caching, error events, number of found updates, and so on.
+
```
Comment (HonkingGoose) [inline docs/usage/opentelemetry.md:24]:
```suggestion
```

**Review by Churro — APPROVED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**